### PR TITLE
CASMINST-2951-1.2 Correct large DNS and FIXME sections which were mistakenly removed

### PR DESCRIPTION
**Pull request by haroldlongley**
## Summary and Scope

CASMINST-2951-1.2 Correct large DNS and FIXME sections which were mistakenly removed

The previous mod to CASMINST-2951 mistakenly removed major DNS and FIXME sections and had used the old dtr.dev.cray.com URL in many places.

### CODE REVIEW

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
This needs to be rebased before it can be reviewed.

**Comment by mharding-hpe** [comment]
@haroldlongley is this PR still needed?

File: install/prepare_site_init.md [hunk 1]
```diff
@@ -133,51 +131,40 @@ with system-specific customizations.
...
+    1. Set environment variables for LDAP and PORT
 
-    ```bash
-    linux# export LDAP=dcldap2.us.cray.com
-    linux# export PORT=636
-    ```
+       In the example below, the LDAP server has the hostname `dcldap2.us.cray.com` and is using the port 636.
+
+       ```bash
+       linux# export LDAP=dcldap2.us.cray.com
```
Comment (rustydb) [inline install/prepare_site_init.md:139]: Can you pretty please remove the `linux#` prompts? If this needs to be ran on an _external_ machine, use `external#` but with the new `main` branch convention: https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#step-examples

**Review by rustydb — CHANGES_REQUESTED** [review]

**Comment by mharding-hpe** [comment]
This PR is no longer needed. Closing it out.

**Pull request closed without merging**